src/Domain/CommentRepository.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace WMDE\Fundraising\Frontend\Domain;
+
+interface CommentRepository {
+
+	/**
+	 * @param int $limit
+	 *
+	 * @return Comment[]
+	 */
+	public function getComments( $limit );
```

Comment:
Eventually there might be. However at this point the use case does not provide pagination yet, so I've not (IMO prematurely) added this to the service is uses either.